### ISSUE TITLE
TSC/TSR: evaluation functionality and interface to losses/metrics

### ISSUE THREAD

**Issue by fkiraly** [alan-turing-institute/sktime#3]
implementation of rudimentary evaluation functionality, should include:
- computation of methods' predictions on test set, after training on training set
- computation of common average and aggregate losses/scores for classification and regression
- interface to metrics/scoring functionality as in sklearn
- at first, preliminary design of the above

Note: *not* full-blown orchestration and experiment management

Conditional on https://github.com/kiraly-group/sktime/issues/2 since calls the estimator interface which needs to be consolidated first.
